### PR TITLE
fopen() vs is_readable()

### PR DESCRIPTION
**Pull request by fglueck**

### CODE REVIEW

File: src/Tag.php [hunk 1]
```diff
@@ -108,21 +108,23 @@ public function __construct(
...
-				$this->otl,
-				$this->tableOfContents,
-				$this->sizeConverter,
-				$this->colorConverter,
-				$this->imageProcessor,
-				$this->languageToFont
-			);
-		}
+	  if (preg_match('/^[a-zA-Z]+$/', $tag)) {
+	    $className = self::getTagClassName($tag);
```
Comment (Thoulah) [inline src/Tag.php]: Please keep indent to tabs: https://github.com/mpdf/mpdf/blob/development/ruleset.xml

**Comment by finwe** [comment]
Thanks. Please, fix the CI, squash middle two commits, and add a unit test reproducing the faulty behaviour for the change with utf8 tag characters.

For changes to be incorporated more quickly, separate different concerns of code to their own pull requests.

File: src/Mpdf.php [hunk 2]
```diff
@@ -9407,7 +9407,7 @@ function Output($name = '', $dest = '')
 						header('Content-Length: ' . strlen($this->buffer));
 					}
 
-					header('Content-disposition: inline; filename="' . $name . '"');
+					header('Content-Disposition: attachment; filename="' . $name . '"; filename*=utf-8\' \'' . rawurlencode($name));
```
Comment (Klap-in) [inline src/Mpdf.php]: Could you give an explanation why this should be changed from inline to attachment?

**Comment by finwe** [comment]
I am going to close this as too broad and stale. @fglueck please, stick with a single concern per PR and always include a unit test that actually covers changes in the code.

**Pull request closed without merging**